src/rules.ts

```diff
@@ -1,9 +1,53 @@
-export interface Rule {
+import {ResourceGraphModels} from '@azure/arm-resourcegraph';
+import {DefaultAzureCredential} from '@azure/identity';
+import {AzureClient} from './azure';
+import {RuleSchema, ScanResult} from './scanner';
+
+export type Rule = ResourceGraphRule;
+
+interface Execute {
```

Comment:
I still think we want some iteration & cleanup around Execute / Rule, but let's land this & address it in a follow-up